garak/resources/autodan/genetic.py

```diff
@@ -31,38 +31,12 @@
     nltk.download("punkt")
     nltk.download("wordnet")
 
-
-class MutationGenerator(OpenAIGenerator):
-    def __init__(self, name, generations=10):
-        """
-        Initialize MutationGenerator class. Superclassing OpenAI Generator to allow multiple messages.
-        Args:
-            name (str): Model name
-            generations (int): Number of generations
-        """
-        super().__init__(name, generations=generations)
-
-    def generate_completion(self, messages: list):
-        response = self.generator.create(
-            model=self.name,
-            messages=messages,
-            temperature=self.temperature,
-            top_p=self.top_p,
-            n=self.generations,
-            stop=self.stop,
-            max_tokens=self.max_tokens,
-            presence_penalty=self.presence_penalty,
-            frequency_penalty=self.frequency_penalty,
-        )
-        return response
-
-
 openai_api_key = os.getenv("OPENAI_API_KEY", default=None)
 if openai_api_key is None:
     USE_OPENAI = False
 else:
     USE_OPENAI = True
-    mutation_generator = MutationGenerator("gpt-3.5-turbo")
+    mutation_generator = OpenAIGenerator("gpt-3.5-turbo")
```

Comment:
is this subtle default going to throw people, do you think?

Reply:
This is the implementation in the paper, but we can set this as a default and allow it to be configurable.